understanding/20/contrast-minimum.html

```diff
@@ -11,14 +11,13 @@ <h1>Understanding Contrast (Minimum)</h1>
    <section id="intent">
       <h2>Intent of Contrast (Minimum)</h2>
       
-      <p>The intent of this Success Criterion is to provide enough contrast between text and
-         its background so that it can be read by people with moderately low vision (who do
-         not use contrast-enhancing assistive technology). For people without color deficiencies,
-         hue and saturation have minimal or no effect on legibility as assessed by reading
-         performance (Knoblauch et al., 1991). Color deficiencies can affect luminance contrast
-         somewhat. Therefore, in the recommendation, the contrast is calculated in such a way
-         that color is not a key factor so that people who have a color vision deficit will
-         also have adequate contrast between the text and the background.
+      <p>The intent of this Success Criterion is to provide enough luminance contrast between text and its background, so that it can be read by people with moderately low vision or impaired contrast perception, without the use of contrast-enhancing assistive technology.
```

Comment:
```suggestion
      <p>The intent of this Success Criterion is to provide enough contrast between text and its background, so that it can be read by people with moderately low vision or impaired contrast perception, without the use of contrast-enhancing assistive technology.
```